README.md

```diff
@@ -69,109 +69,115 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Code analyzers have been upgraded!  Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
-- Symbols are back!
-- Code refactoring, thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper) and [@cyberrex5](https://github.com/cyberrex5) for helping here!
-- We are now on VS 2022 with the .NET 6 upgrade.
+- OOBE code refactor to have all module information in XAML, like in Settings. Thanks [@niels9001](https://github.com/niels9001)!
+- Every project now has analyzers turned on and warnings fixed. Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
+- New patterns added for code spell-checking and stale entries removed. Thanks [@jsoref](https://github.com/jsoref)
+- Additional logging has been added to Fancy Zones and PowerToys Run.
+- A new CI release build will not be triggered if all that was changed was just documentation.
+- Fixed a racing condition causing flaky build errors when building Power Rename.
+- Centralization of common csproj/vcxproj settings underway. Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
 
 #### Community contributions
 
-We'd like to directly mention certain contributors (in alphabetical order) for their continued community support this month and helping directly make PowerToys a better piece of software. 
-[@Aaron-Junker](https://github.com/Aaron-Junker), [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper), [@cyberrex5](https://github.com/cyberrex5), [@franky920920](https://github.com/franky920920), [@gsuberland](https://github.com/gsuberland), [@htcfreek](https://github.com/htcfreek), [@jay-o-way](https://github.com/jay-o-way), [@makeProjectGreatAgain](https://github.com/makeProjectGreatAgain), [@niels9001](https://github.com/niels9001), [@robmen](https://github.com/robmen), [@snickler](https://github.com/snickler), and [@TobiasSekan ](https://github.com/TobiasSekan).
+We'd like to directly mention certain contributors (in alphabetical order) for their continued community support this month and helping directly make PowerToys a better piece of software.  
+[@Aaron-Junker](https://github.com/Aaron-Junker), [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper), [@dend](https://github.com/dend), [@franky920920](https://github.com/franky920920), [@htcfreek](https://github.com/htcfreek), [@jay-o-way](https://github.com/jay-o-way), [@jsoref](https://github.com/jsoref), [@niels9001](https://github.com/niels9001), [@snickler](https://github.com/snickler).
 
 
-#### What is being planned for v0.57
+#### What is being planned for v0.58
 
-For [v0.57][github-next-release-work], we'll start work on below:
+For [v0.58][github-next-release-work], we'll start work on below:
 
-- Start work on two new PowerToys
-- Improvements to PowerToy Run plugins
+- Environment Variables Editor PowerToy
+- Continue work on another new PowerToy
 - Stability / bug fixes
-- Validation pass again using WinUI 3.1 for Settings
-- Adding new file types to dev file preview
```

Comment:
Can be readded as this is under development.

cc: @Aaron-Junker

Reply:
You're right

Reply:
Re-adding the new file types to dev file preview